phalcon/mvc/model/query.zep

```diff
@@ -2513,6 +2522,93 @@ class Query implements QueryInterface, InjectionAwareInterface
 	}
 
 	/**
+	 * gets the read connection from the model if there no transaction inside the query object
```

Comment:
but they didn't exist before and are not in the codebase till now! :) 

so there is no compatibility change. and all the other methods are still compatible because i kept the transaction parameter optional.

so is that an "okay" ? or is that TMI and I just show you the endproduct because you got better things todo ? ... just didn't want to piss you off :)